src/components/icons/HeartIcon.tsx

```diff
@@ -1,18 +1,25 @@
 import { IconProps } from "../../types";
 
-export const HeartIcon = ({ pathClassName, ...props }: IconProps) => {
+type HeartIconProps = {
+  isFav: boolean;
+} & IconProps;
+
+export const HeartIcon = ({
+  pathClassName,
+  isFav,
+  ...props
+}: HeartIconProps) => {
   return (
     <svg
       width="20"
       height="20"
       viewBox="0 0 20 20"
-      fill="none"
       xmlns="http://www.w3.org/2000/svg"
       {...props}
     >
       <path
         className={pathClassName}
-        fill-rule="evenodd"
+        fill-rule={isFav ? "" : "evenodd"}
```

Comment:
suggestion: for this case `fillRule={!isFav && "evenodd"} ` as it is more concise